bin/copy-labels.js

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const util = require('util');
+
+const { LabelHandler } = require('../lib/labels.js');
+
+const toolName = path.basename(process.argv[1]);
+const toolVersion = require('../package.json').version;
+
+const padding = ' '.repeat(toolName.length);
+
+const help = `\
+This tool copies labels from source repository or file to destination
+repository or file.
+
+Usage: ${toolName} [--config=path] [--user=user] [--token=token]
+       ${padding} [--src-repo=repo] [--dst-repo=repo] [--src-file=path]
+       ${padding} [--dst-file=path] [--delete] [--update]
+       ${toolName} --help
+       ${toolName} --version
+
+Options:
+  --config   Path to config file
+  --user     github username
+  --token    github access token
+  --src-repo Source repository. e.g. 'metarhia/tools'
```

Comment:
Instead of introducing four options for `src` and `dst` and since both of them are not optional, we can just move them to arguments, so that it would be used in the following way (similar to Unix `cp` tool):
```
Usage: toolname [OPTION]... SOURCE DEST
```

Reply:
This is nice, but how you distinguish `metarhia/tools` repo and `metarhia/tools` file? Although we can demand that file should always have `.json` extension. Wdyt?

Reply:
Yes, I was assuming that files will be identified by the extension.